staging/src/k8s.io/client-go/transport/round_trippers.go

```diff
@@ -33,20 +33,18 @@ import (
 // HTTP2 clients). Pure HTTP clients should use the RoundTripper returned from
 // New.
 func HTTPWrappersForConfig(config *Config, rt http.RoundTripper) (http.RoundTripper, error) {
-	if config.WrapTransport != nil {
```

Comment:
not dropped, it's part of the auth provider, moved to here: https://github.com/kubernetes/kubernetes/pull/45817/files#diff-f5423923c5f61f08517f3eb0014e48eeR439